src/Views/Form/Templates/Sequoia/assets/js/form.js

```diff
@@ -415,8 +415,11 @@
 		setupSelectInputs();
 	}
 
+	/**
+	 * Limited scope of optional input labels, specifically to User Info, see issue #5160.
+	 */
 	setupOptionalInputLables(
```

Comment:
@henryholtgeerts I find out that there is a typo in the function name. function label will not re-setup when payment gateway field reload when switching payment gateway. Can you confirm and reopen this issue?

or maybe we can resolve this with https://github.com/impress-org/givewp/issues/4944

cc: @kjohnson

Reply:
@ravinderk Let's correct this in a PR to address #4944. As it is currently, the code that invokes this function, also uses the typo. You are right that the function is only being called on page load.

Reply:
I am realizing now, that because fields could be placed within the payment gateway area (per setting intended for the legacy forms) it is critical that styles and handlers which are initialized on page load, should also be reset on each payment gateway change. This is accounted for in fields add by addons (ex: FFM fields), but I am unsure of how many core fields are not being reset on each payment gateway change.